Example/Backpack/CustomPresentable.swift

```diff
@@ -0,0 +1,26 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2021 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+struct CustomPresentable: Presentable {
```

Comment:
I wonder if there's a better name for this? Could also be nice to have some docs. For clarity maybe we should put all the `Presentable` related code in a single file called `Presentable.swift`?